tests/storage-vm

```diff
@@ -484,6 +484,35 @@ for poolDriver in $poolDriverList; do
 
         lxc delete -f v1
 
+        echo "==> Checking setting instances.migration.stateful for VMs"
```

Comment:
@gabrielmougard @simondeziel oh I just merged this, but then reaslised its not checking the API extension and skipping this test if server doesn't support it. Please can you update in a new PR or the 5.0/edge tests will fail. Thanks @gabrielmougard

Reply:
@tomponline https://github.com/canonical/lxd-ci/pull/84